src/nar/serialize.go

```diff
@@ -1,10 +1,15 @@
 package nar
 
 import (
+	"encoding/binary"
 	"fmt"
 	"io"
 )
 
+const (
+	MaxInt64 = 1<<(64-1) - 1
```

Comment:
`math.MaxInt64` also has that value. But maybe you didn't want to import the whole module just for the constant.

Reply:
I think the go compiler is pretty good in dead code elimination. I'll use `math.MaxInt64`, thanks!